Snowdog_Components/components/Templates/multishipping/select-address/select-address.config.js

```diff
@@ -58,17 +58,73 @@ module.exports = {
                   {
                     childTag: 'td',
                     childTagAttributes: 'data-th="Qty:" role="gridcell"',
-                    contentComponent: 'quantity-update--full'
+                    contentComponent: 'quantity-update--full',
+                    contentContext: {
+                      class: 'quantity-update--no-buttons-m',
+                      qtyClass: 'input',
+                      qtyLabel: 'Qty',
+                      input: {
+                        id: 'qty',
+                        min: '1',
+                        name: 'qty',
+                        defaultValue: '1',
+                        ariaLabel: 'Change the quantity'
+                      },
+                      minusQtyButton: {
+                        tag: 'button',
+                        class: 'quantity-update__button quantity-update__button--minus',
+                        attributes: 'type="button" aria-label="Decrease the quantity"',
+                        icon: {
+                          id: 'minus',
+                          title: 'Minus mark',
+                          class: 'button__icon quantity-update__icon',
+                          hidden: true
```

Comment:
> When you set `hidden` value in config context to `true` role will change to `presentation`, an attribute `focusable="false"` will be added and icon won't be visible for assistive technologies (AT).

Title is only required for icons with `role="img"`.
Can you remove `title` in configs from icons with role presentation?